core/src/main/java/org/apache/carbondata/core/util/CarbonMetadataUtil.java

```diff
@@ -521,6 +521,8 @@ private static int compareMeasureData(byte[] first, byte[] second, DataType data
         compare = -1;
       }
       return (int) compare;
+    } else if (dataType == DataTypes.BINARY) {
+      return 0;
```

Comment:
Binary is a dimension. These changes are in Measure flow. please remove this change

Reply:
ok, done